### ISSUE TITLE
Question about loss on valid image region during training 

### ISSUE THREAD

**Issue by JoyHuYY1412** [adobe-research/custom-diffusion#90]
Hi Nupur, thank you for your great work. My question is as you mentioned you only "backpropagate the loss on valid regions", and I found you use mask to mask out the loss as `loss = ((loss*mask).sum([1, 2, 3])/mask.sum([1, 2, 3])).mean()`. Does here the image mask means valid regions in your paper? And you only want to reconstruct the object so that it won't overfit to the background, am I right?

**Comment by nupurkmr9** [comment]
Hi, in our method, we perform some basic data augmentations like resizing the image and copy-pasting it on a larger canvas; in this case, the valid region is the smaller copy-pasted image. 
This idea can be extended to only reconstruct the object as well and ignore the background. However, in my observation, having different backgrounds helps the model learn the concept better. 
Hope this answers your question. 

Thanks.
